kubernetes/cache_test.go

```diff
@@ -0,0 +1,166 @@
+package kubernetes
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
this github.com third party lib import should be in the second section of the imports to match coding style.